HelpSource/Classes/Pbind.schelp

```diff
@@ -89,6 +89,42 @@ Pbind(*[
 )
 ::
 
+
+subsection::Multiple Instruments
+
+Pbind cannot be used to play multiple instruments. The following will not work.
+
+code::
+Pbind(
+     \instrument, [\beep, \boop], // this line is wrong
+     \dur, 1,
+     \freq, 220
+ )
+::
+
+This is a limit of the implmentation, but also encourages the use of smaller more managable patterns.
+
+To help solve this the following function is provided, but it is just one way to solve the problem - feel free to copy it into your projects as needed!
+
+code::
+~putInstrumentsInPpar = {|...instruments|
+	Ppar(instruments.collect({|i| Pbind(\instruments, i) }), inf)
```

Comment:
There is a secret convention that `i` is the counting argument. Better use `item`, `each`, or `x`.